easybuild/easyconfigs/r/Reapr/Reapr-1.0.18-cleanup.patch

```diff
@@ -0,0 +1,219 @@
+This patch makes Reapr compatible with EasyBuild.
+
+Henrik R. Nagel, 20191115
+diff -ruN Reapr_1.0.18.orig/Makefile Reapr_1.0.18/Makefile
+--- Reapr_1.0.18.orig/Makefile	1970-01-01 01:00:00.000000000 +0100
++++ Reapr_1.0.18/Makefile	2019-11-13 12:17:03.000000000 +0100
+@@ -0,0 +1,8 @@
++all:
++	cd third_party/tabix ; make
```

Comment:
`tabix` is provided by `HTSlib`, isn't adding that as a dependency a better alternative?

Reply:
The problem is this line in src/Makefile:
``TABIX = tabix/tabix.o  -L./tabix -ltabix -lz``
Neither the tabix easyconfig nor the HTSLib easyconfig allows linking with the static tabix library file that is created during compilation. I don't know why the Makefile also links with tabix.o.

Reply:
@boegel thoughts on how to resolve this one?

Reply:
I overlooked the explicit linking to `tabix.o`...

I can't think of a reason why that would be needed when also linking to `-ltabix`, but I guess this approach is the easiest way forward. It's not like `tabix` is a huge dependency, or that taking this approach would cause any major problems.

Plus, there's a whole bunch of (useful) patching going on here already, so not including this bit in the patch wouldn't prevent the need for a patch anyway.